Add doc dir for Keptn 0.16 (#1228)
keptn/keptn.github.io: content/docs/0.16.x/continuous_delivery/delivery_assistant/index.md

    - name: "delivery"
      triggeredOn:
        - event: "dev.delivery.finished"
      tasks:
        - name: "approval"
          properties:
            pass: "manual"
            warning: "manual"
        - name: "deployment"
          properties:
            deploymentstrategy: "blue_green_service"
        - name: "test"
          properties:
            teststrategy: "performance"
        - name: "evaluation"
        - name: "release"
```

If this task is in place, you can control how to proceed with the delivery depending on the evaluation result from the previous stage. You have the options to configure this as follows: 

| Evaluation result                 | Option: `manual`                                  | Option: `automatic`                 |
|---------------------------------  |-----------------------------------------------  |---------------------------------- |
| If evaluation returned is `pass`      | `manual` - the user has to manually approve/decline it.   | `automatic`- Keptn automatically approves it.   |
| If evaluation returned is `warning`   | `manual` - the user has to manually approve/decline it.   | `automatic` - Keptn automatically approves it.  |

> *Example - Regardless of the evaluation result, the user needs to approve or decline it:*

Reviewer: ```suggestion
*Example - Regardless of the evaluation result, the user must approve or decline it:*
```

      caption="Keptn Bridge Assisted Delivery">}}

## Approve or decline an open approval with Keptn CLI

* To fetch a list of open approvals, Keptn CLI provides the [`keptn get event approval.triggered`](../../reference/cli/commands/keptn_get_event_approval.triggered/) command:

    ```console
    $ keptn get event approval.triggered --project=sockshop --stage=production
    Starting to get approval.triggered events
    {
        "contenttype": "application/json",
        "data": {
            "deploymentstrategy": "blue_green_service",
            "image": "docker.io/keptnexamples/carts",
            "labels": null,
            "project": "sockshop",
            "result": "warning",
            "service": "carts",
            "stage": "production",
            "tag": "0.13.2",
            "teststrategy": "performance"
        },
        "id": "5bf26759-4afa-4045-8ccf-81bc398c2fcd",
        "shkeptncontext": "654c80b0-4a02-4d36-96f3-7447df1cdf41",
        "source": "gatekeeper-service",
        "specversion": "1.0",
        "time": "2020-09-17T14:41:14.039Z",
        "type": "sh.keptn.event.approval.triggered"
    }
    ```

> * In addition, by using the command [`keptn send-event approval.finished`](../../reference/cli/commands/keptn_send_event_approval.finished/) the CLI will print a list of open approvals and allows you to choose which one you want to approve/decline:

Reviewer: ```suggestion
* In addition, by using the command [`keptn send-event approval.finished`](../../reference/cli/commands/keptn_send_event_approval.finished/) the CLI prints a list of open approvals and allows you to choose which one you want to approve/decline:
```
